src/main/java/Strategy/ConcreteAutoGameInitStrategy.java

```diff
@@ -6,15 +6,92 @@
 import model.ConnectorType;
 
 import java.util.HashMap;
+import java.util.HashSet;
 
 public class ConcreteAutoGameInitStrategy implements GameInitStrategy {
     @Override
-    public GameBuilder initializeGame(GameBuilder builder, ValidatorFacade validatorFacade) {
+    public GameBuilder initializeGame(GameBuilder gameBuilder, ValidatorFacade validatorFacade) {
+
+        gameBuilder.setLadders(initializeLadders(gameBuilder, validatorFacade));
+        gameBuilder.setSnakes(initializeSnakes(gameBuilder, validatorFacade));
+
+        return gameBuilder;
+    }
+
+    @Override
+    public HashMap<Integer, Connector> initializeSnakes(GameBuilder gameBuilder, ValidatorFacade validatorFacade) {
```

Comment:
The method initializeSnakes can be private, if it is used by all subclasses as an exact implementation it can be added to the GameInitStrategyInterface.